workspaces/cli-server/src/routers/spec-router.ts

```diff
@@ -239,6 +239,7 @@ ${events.map((x: any) => JSON.stringify(x)).join('\n,')}
         const events = DiffEngine.commit(commands, {
           specDirPath: req.optic.paths.specDirPath,
           commitMessage: payload.commitMessage,
+          appendToRoot: !isEnvTrue(process.env.OPTIC_SPLIT_SPEC_EVENTS),
```

Comment:
Using the flag here to keep reading from the env as close to the edge of the system as possible.